EKF/ekf_helper.cpp

```diff
@@ -719,6 +719,8 @@ bool Ekf::setEkfGlobalOrigin(const double latitude, const double longitude, cons
 		map_projection_reproject(&_pos_ref, _state.pos(0), _state.pos(1), &current_lat, &current_lon);
 		current_alt = -_state.pos(2) + _gps_alt_ref;
 		current_pos_available = true;
+	} else {
+		_NED_origin_initialised = true;
```

Comment:
I wouldn't set the flag here because the meaning of that `else` is: "if the map is not already initialized" and not if we actually initialized the map with the new origin.
I would suggest to place it above the `return true` below, as this is the place where we know that the map has been initialized correctly with the new origin.